packages/webdriverjs/README.md

```diff
@@ -36,140 +36,115 @@ driver.get('https://dequeuniversity.com/demo/mars/').then(function () {
 });
 ```
 
-### AxeBuilder(driver:WebDriver[, axeSource:string])
+## AxeBuilder(driver: Webdriver.WebDriver[, axeSource: string])
 
-Constructor for the AxeBuilder helper. You must pass an instance of selenium-webdriver as the first and only argument.
+Constructor for the AxeBuilder helper. You must pass an instance of WebdriverJS as the first argument.
 
-```javascript
-var builder = new AxeBuilder(driver);
+```js
+const builder = new AxeBuilder(driver);
 ```
 
-If you wish to run a specific version of axe-core, you can pass the source axe-core source file in as a string. Doing so will mean axe-webdriverjs runs this version of axe-core, instead of the one installed as a dependency of axe-webdriverjs.
+If you wish to run a specific version of [axe-core](https://github.com/dequelabs/axe-core), you can pass the source of axe-core source file in as a string. Doing so will mean `@axe-core/webdriverjs` run this version of axe-core, instead of the one installed as a dependency of `@axe-core/webdriverjs`.
 
-```javascript
-var axeSource = fs.readFileSync('./axe-1.0.js', 'utf8');
-var builder = new AxeBuilder(driver, axeSource);
+```js
+const axeSource = fs.readFileSync('./axe-1.0.js', { encoding: 'utf-8' });
```

Comment:
Why specify the encoding like this? Passing a string is _much_ more common here.

```suggestion
const axeSource = fs.readFileSync('./axe-1.0.js', 'utf8');
```